CRM/Contribute/Import/Parser/Contribution.php

```diff
@@ -593,8 +593,9 @@ public function formatDateFields(&$params) {
    * CRM_Contribute_Import_Parser_ContributionTest.
    *
    * @param array $params
+   * @param array $formatted
    */
-  public function formatInput(&$params) {
+  public function formatInput(&$params, &$formatted) {
```

Comment:
I think this will solve the test failure

```suggestion
  public function formatInput(&$params, &$formatted = []) {
```